lib/Installation/Partitioner/LibstorageNG/v4/ExpertPartitionerController.pm

```diff
@@ -133,6 +137,37 @@ sub clone_partition_table {
     $self->get_expert_partitioner_page()->press_ok_clone();
 }
 
+sub create_msdos_partition_table {
+    my ($self) = @_;
+    $self->get_expert_partitioner_page()->go_top_in_system_view_table();
```

Comment:
why you go top again here if `run_expert_partitioner()` finishes with focus in the expanded hard disks? I think we just need to click on "Modify" button and and on its menu item "Create new Partition Table..."

Reply:
I am not sure if I understand what you mean. In order for the "modify" button to appear, the "Hard Disks" must be selected. I don't think that's occurring at the end of expert partitioner.